app/controllers/maps_controller.rb

```diff
@@ -86,6 +86,20 @@ def usermaps
     end
   end
 
+  # GET maps/new
+  def new
+    @map = Map.new(:name => "Untitled Map", :permission => "public", :arranged => true)
```

Comment:
Because I think it might not be right to make the default setting for maps
that people create ones in which anyone at all with an account can edit
their map. This has worked well when we have so few users, but it doesn't
bode well with more people signing on. I think we have to give them a
better clue how to
1. Invite people to collaborate on their map
2. Better display the option to convert their map to Commons, and why

Along with that, it would be great to be able to revert your Commons map
back to before someone accidentally or intentionally removed everything
from it

On Aug 15, 2016 8:47 PM, "Devin Howard" <notifications@github.com> wrote:

> In app/controllers/maps_controller.rb
> <https://github.com/metamaps/metamaps/pull/607#discussion_r74861287>:
>
> > @@ -86,6 +86,20 @@ def usermaps
> >      end
> >    end
> >
> > +  # GET maps/new
> > +  def new
> > +    @map = Map.new(:name => "Untitled Map", :permission => "public", :arranged => true)
>
> In new ruby you can rewrite to
>
> @map = Map.new(name: "Untitled Map", permission: "public", arranged: true)
>
> Hold on!! Any reason you went with public and not commons?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/metamaps/metamaps/pull/607/files/2c54d906fc5df57ae9f727c0ca1ecce281a817f5#r74861287>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABWAYSZVb53O692sB0Dy4iFqXZh2rVN3ks5qgQiigaJpZM4Jkugb>
> .
>